cluster/images/etcd/Makefile

```diff
@@ -41,6 +41,9 @@ build:
 	# Copy the content in this dir to the temp dir
 	cp ./* $(TEMP_DIR)
 
+	make -C ../../../ WHAT=cluster/images/etcd KUBE_STATIC_OVERRIDES="etcd";
+	cp "../../../_output/bin/etcd" $(TEMP_DIR)/attachlease
```

Comment:
Actually no.
The problem is that "make build" when it compiles a binary, it calls it with a name of a directory (which in this case is "etcd".
However, I can move it to attachlease subdirectory - this will probably be more naturaal. Will fix.